code/src/components/ClearButton.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { useDispatch } from "react-redux"
+import { todoTasks } from 'reducers/todoTasks'
+
+export const ClearButton = () => {
+    const dispatch = useDispatch()
+
+    return (
+        <button 
+            className="clearButton" 
+            type = 'button' onClick= {() => dispatch(todoTasks.actions.removeAll())}>
```

Comment:
Props and html attributes should not have a space around the `=`. Also, when you start putting the attributes on new lines like this, stick with it and put each one on a new line. So this would be:

```
<button
  className="clearButton"
  type="button"
  onClick={() => dispatch(todoTasks.actions.removeAll())}>
  Done & clear
</button>
```